library/general/src/lib/cfa/sysctl_config.rb

```diff
@@ -0,0 +1,211 @@
+            "But they will be overruled be manual setting described in %{file_list}."), yast_file_name: YAST_CONFIG_PATH, file_list: conflict_files.join(", ")))
+        end
+      end
+      yast_config_file&.save
+    end
+
+    # Whether there is a conflict with given attributes
+    #
+    # @param only [Array<Symbol>] attributes to check
+    # @return [Array<String>] list of conflicting files
+    def conflict_files(only: [])
+      return [] if yast_config_file.empty?
+
+      conflicting_attrs = yast_config_file.present_attributes
+      if !only.empty?
+        # Dropping all not needed values
+        conflicting_attrs.each_key do |key|
+          conflicting_attrs.delete(key) unless only.include?(key)
+        end
+      end
+      file_list = []
+      higher_precedence_files.each do |file|
+        # Checking all "higher" files if their values overrule the current
+        # YAST settings.
+        higher_attr = file.present_attributes
+        file_list << file.file_path if conflicting_attrs.any? { |k, v| !higher_attr[k].nil? && v != higher_attr[k] }
+      end
+      file_list
+    end
+
+    def files
+      @files ||= config_paths.map { |file| Sysctl.new(file_path: file) }
+    end
+
+  private
+
+    def yast_config_file
+      @yast_config_file ||= files.find { |f| f.file_path == YAST_CONFIG_PATH }
+    end
+
+    def lower_precedence_files
+      @lower_precedence_files ||= files[0...yast_config_file_idx]
+    end
+
+    def higher_precedence_files
+      @higher_precedence_files ||= files[(yast_config_file_idx + 1)..-1]
+    end
+
+    def config_paths
+      paths = PATHS.each_with_object([YAST_CONFIG_PATH]) do |path, all|
+        all.concat(file_paths_in(path))
+      end
+
+      paths.uniq! { |f| File.basename(f) }
+      # Sort files lexicographic
+      paths.sort_by! { |f| File.basename(f) }
+
+      # Prepend the kernel configuration file
+      paths.unshift(boot_config_path) unless boot_config_path.empty?
+
+      paths
+    end
+
+    def file_paths_in(path)
+      if Yast::FileUtils.IsFile(path)
+        [path]
+      elsif Yast::FileUtils.IsDirectory(path)
+        Yast::SCR.Read(Yast::Path.new(".target.dir"), path).map { |file| File.join(path, file) }
+      else
+        log.debug("Ignoring not valid path: #{path}")
+
+        []
+      end
+    end
+
+    def boot_config_path
+      return @boot_config_path if @boot_config_path
+
+      @boot_config_path = if !kernel_version.empty?
+        "/boot/sysctl.conf-#{kernel_version}"
+      else
+        ""
+      end
+    end
+
+    def boot_config_file
+      @boot_config_file ||= files.find { |f| f.file_path == boot_config_path }
+    end
+
+    def kernel_version
+      @kernel_version ||= Yast::Execute.on_target.stdout("/usr/bin/uname", "-r").to_s.chomp
+    end
+
+    def yast_config_file_idx
+      @yast_config_file_idx ||= files.find_index { |f| f == yast_config_file }
+    end
+
+    def yast_defined_attrs(attrs)
```

Comment:
Is this method really needed?

Reply:
No, you are right. It is a left over of your PR and I have not noticed that it will not be used anymore. :-)